pages/index.js

```diff
@@ -19,6 +19,9 @@ export default function HomePage() {
       >
         <small>Github</small>
       </Link>
+      <Link style={{ color: "white" }} href={"form-and-list"}>
+        <small>User Story 1 – Form and List</small>
+      </Link>
     </div>
```

Comment:
```suggestion
      <Link href={"form-and-list"}/>
    </div>
```
no inline styles and dont include any hints about userstories in your actual code please :) This is just a guide for you as devs.